plugins/jobtype/src/azkaban/jobtype/HadoopSparkJob.java

```diff
@@ -525,7 +525,30 @@ private static void handleStandardArgument(Props jobProps,
       }
     }
 
-    return new String[]{sparkHome, sparkConf};
+    return new String[]{getSparkLibDir(sparkHome), sparkConf};
```

Comment:
The method is still called  getSparkHomeConf and if we return the lib dir here, we should make corresponding name change